rllib/tests/run_regression_tests.py

```diff
@@ -38,7 +38,7 @@
     type=str,
```

Comment:
Since the test logic is looping through all the yaml files in a yaml dir, does it make sense to further split them to have one yaml file per test?